includes/admin/import/class-batch-import-downloads.php

```diff
@@ -310,6 +310,49 @@ private function set_price( $download_id = 0, $price = '' ) {
 
 	}
 
+	/**
+	 * Convert a files string containing delimiters to an array
+	 *
+	 * @since 2.9.20
+	 * @param $str Input string to convert to an array
+	 * @return array
+	 */
+	public function convert_file_string_to_array( $str = '' ) {
```

Comment:
@pippinsplugins Did you already consider putting this in the parent class EDD_Batch_Import?

Reply:
I'm just thinking it might be useful for other extenders as well in the future.

Reply:
I placed it in the download import class because it was specific to products. I don't think core has a use for it in the base class, but someone else that extends it certainly might.

I'll move it.